remote-access/ip-address.md

```diff
@@ -79,6 +79,54 @@ Nmap done: 256 IP addresses (4 hosts up) scanned in 2.41 seconds
 
 Here you can see a device with hostname `raspberrypi` has IP address `192.168.1.8`.
 
+### Getting IPv6 addresses by pinging from a second device (linux device or raspberry pi)
+
+First find your own IP address, in other words the one of the computer you're using to find your Pi's IP address
+by `hostname -I`
+
+`fd00::ba27:ebff:feb6:f293 2001:db8:494:9d01:ba27:ebff:feb6:f293`
```

Comment:
I've just noticed that there's two IP addresses here. :eyes: I've never used IPv6 myself, but how does a user know which IP address to use with the `ping` command below?

Reply:
I will explain it.